llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -175,6 +175,13 @@ static cl::opt<bool> EmitExportedSymbols{"emit-exported-symbols",
                                          cl::desc("emit exported symbols"),
                                          cl::cat(PostLinkCat)};
 
+static cl::opt<bool> EmitNonKernelEntryPoints{
+    "emit-non-kernel-entry-points",
```

Comment:
I would suggest to reverse the semantic of the option, as "normal" behavior is to treat SYCL external functions just like kernels. "emit-only-kernel-entries" with default 0.

Reply:
Yes, that makes sense, applied in 4cdb8300442685ab045a47e2e34127586714fc0e